client-super-turtle-tracker/src/components/reusables/TurtleList.tsx

```diff
@@ -0,0 +1,26 @@
+import useAxios from "axios-hooks";
+import * as React from "react";
+import { ScrollView } from "react-native";
+import { View } from ".";
+import { BACKEND_SECRET, BASE_URL } from "../../../env";
+import { Turtle } from "../../utils/interfaces/Turtle";
+import { TurtleListItem } from "./TurtleListItem";
+
+export const TurtleList = (): JSX.Element => {
+  const [{ data, loading, error }, refetch] = useAxios({
+    headers: { Authorization: `Bearer ${BACKEND_SECRET}` },
+    url: `${BASE_URL}/turtle`,
+  });
+
+  return (
+    <View>
+      {loading || error ? undefined : (
+        <ScrollView>
+          {data.map((item: Turtle) => {
+            return <TurtleListItem key={item.id} turtle={item} />;
+          })}
+        </ScrollView>
```

Comment:
Inside the `<ScrollView>` would you be able to implement a `<FlatList>`? It helps to increase performance by only rendering elements on the screen. See this for more explanation and examples: https://reactnative.dev/docs/using-a-listview

Reply:
Actually, you might remove <ScrollView>

Reply:
I tried to implement a `<Flatlist>` but I don't think it's compatible with the `<List.Item>` component from `react-native-paper`. Maybe I'm wrong and you can help me figure it out tomorrow, but I'm getting a bunch of weird type errors that I can't seem to figure out.

Reply:
nvm. I have it mostly fixed now

Reply:
So now it works, but I have 1 es-lint error that I couldn't fix. I'm pretty confused about it, so maybe we can look at it together tomorrow